rdmo/__init__.py

```diff
@@ -1 +1,7 @@
-__version__ = "2.4.0"
+from importlib.metadata import PackageNotFoundError
+from importlib.metadata import version as _version
+
+try:
+    VERSION = __version__ = _version(__package__)
```

Comment:
Last question: Which part of the project consumes `VERSION`? I saw a couple of usages of `___version__`.